test/wallet-test.js

```diff
@@ -1,5 +1,3 @@
-/* eslint-env mocha */
-/* eslint prefer-arrow-callback: "off" */
 /* eslint no-implicit-coercion: "off" */
```

Comment:
we could lose this rule too and remove the `1 *` here: 

https://github.com/handshake-org/hsd/blob/eea15c2f2d6b3371eefb3bb7a93d08185bc9e25f/test/wallet-test.js#L2466-L2467

although I think that makes the test easier to understand, since that integer gets bumped to `2` in the next test

Reply:
Can be fixed by adding this commit:

```diff
diff --git a/test/wallet-test.js b/test/wallet-test.js
index dcf17b540..1f12fafc7 100644
--- a/test/wallet-test.js
+++ b/test/wallet-test.js
@@ -1,5 +1,3 @@
-/* eslint no-implicit-coercion: "off" */
-
 'use strict';
 
 const assert = require('bsert');
@@ -2461,8 +2459,8 @@ describe('Wallet', function() {
       let bal = await wallet.getBalance();
       assert.strictEqual(bal.tx, 2);
       assert.strictEqual(bal.coin, 2);
-      assert.strictEqual(bal.confirmed, 10e6 - (1 * fee));
-      assert.strictEqual(bal.unconfirmed, 10e6 - (1 * fee));
+      assert.strictEqual(bal.confirmed, fund - (cTXCount * fee));
+      assert.strictEqual(bal.unconfirmed, fund - (uTXCount * fee));
       assert.strictEqual(bal.ulocked, 0);
       assert.strictEqual(bal.clocked, 0);
 
@@ -2496,8 +2494,8 @@ describe('Wallet', function() {
       let bal = await wallet.getBalance();
       assert.strictEqual(bal.tx, 3);
       assert.strictEqual(bal.coin, 3);
-      assert.strictEqual(bal.confirmed, 10e6 - (2 * fee));
-      assert.strictEqual(bal.unconfirmed, 10e6 - (2 * fee));
+      assert.strictEqual(bal.confirmed, fund - (cTXCount * fee));
+      assert.strictEqual(bal.unconfirmed, fund - (uTXCount * fee));
       assert.strictEqual(bal.ulocked, lockup);
       assert.strictEqual(bal.clocked, lockup);
 
@@ -2557,8 +2555,8 @@ describe('Wallet', function() {
       let bal = await wallet.getBalance();
       assert.strictEqual(bal.tx, 4);
       assert.strictEqual(bal.coin, 4);
-      assert.strictEqual(bal.confirmed, 10e6 - (3 * fee));
-      assert.strictEqual(bal.unconfirmed, 10e6 - (3 * fee));
+      assert.strictEqual(bal.confirmed, fund - (cTXCount * fee));
+      assert.strictEqual(bal.unconfirmed, fund - (uTXCount * fee));
       assert.strictEqual(bal.ulocked, value);
       assert.strictEqual(bal.clocked, value);
 

```